### PR TITLE
Fix broken links for linkinator check

### PR DESCRIPTION
**Pull request by WaleedAshraf**
Identified in separate PR: https://github.com/openjs-foundation/cross-project-council/pull/1222

### CODE REVIEW

File: meetings/2021/2021-01-05.md [hunk 1]
```diff
@@ -54,11 +54,10 @@
 
 * Improving Diversity and Inclusion at the OpenJSF
 [#699](https://github.com/openjs-foundation/cross-project-council/issues/699)
-  * Link shared by Robin: https://openjsf.org/blog/2020/12/14/training-course-on-diversity-in-open-source
+  * Link shared by Robin: https://openjsf.org/blog/training-course-on-diversity-in-open-source
   * Link shared by Robin: https://inclusivenaming.org
   * Prompt from Tierney: do we have materials to give to member companies which could also be used by non-member companies to show how leadership provides value in open source projects; specifically around URM being able to show their work/effort/participation
   * Robin will pull data from LF Insights
-  * Brian shared: https://insights.lfx.linuxfoundation.org/projects/ojsf-f
```
Comment (WaleedAshraf) [inline meetings/2021/2021-01-05.md:61]: Can someone verify that the OJSF project was removed from the LF Insights portal? I couldn't find it. cc @bensternthal

Comment (ljharb) [reply meetings/2021/2021-01-05.md:61]: it definitely doesn't seem to be on https://insights.lfx.linuxfoundation.org

File: meetings/2021/2021-04-13.md [hunk 2]
```diff
@@ -22,7 +22,7 @@
 ## Agenda
 
 ### Announcements
-LiFT Scholarships are available now: https://openjsf.org/blog/2021/04/13/lift-scholarship/
+LiFT Scholarships are available now: https://openjsf.org/blog/lift-scholarship
```
Comment (ljharb) [inline meetings/2021/2021-04-13.md:25]: @kyliewd

File: proposals/approved/CPC_CHARTER/CPC-CHARTER.md [hunk 3]
```diff
@@ -309,7 +309,7 @@ repository.
 in an official decision making role as defined in the Project's governance
 policies.
 
-[Foundation bylaws]: https://openjsf.org/wp-content/uploads/sites/84/2020/07/OpenJS-Foundation-Bylaws-2020-06-08.pdf
+[Foundation bylaws]: https://bylaws.openjsf.org
```
Comment (ljharb) [inline proposals/approved/CPC_CHARTER/CPC-CHARTER.md:312]: @kyliewd this should presumably be a 301 redirect

**Review by ljharb — APPROVED** [review]

**Comment by ljharb** [comment]
requesting fast track

**Review by ovflowd — APPROVED** [review]

**Comment by ovflowd** [comment]
Merging this for now, and let's have @kyliewd address the comments after she's back from Holidays.

**Pull request merged**